### PR TITLE
VertexColor/HeightMap, ScaledVersion

### PR DESCRIPTION
**Pull request by DeltaDizzy**

### CODE REVIEW

File: main/ScaledVersion/OnDemand.md [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+subtitle: Quite un-demanding, if you ask me.
 ---
+
+The `OnDemand { }` subnode of the `ScaledVersion { }` node tells Kopernicus to use OnDemand loading for the textures specified inside the subnode. The fields set inside this subnode are exactly the same as their equivalents in the [`Material { }` subnode]({{ "main/ScaledVersion/Material.html" | relative_url }}) of the `ScaledVersion { }` node.
+
+**Example**
+```
+ScaledVersion
+{
+	...
```
Comment (StollD) [inline main/ScaledVersion/OnDemand.md]: Those code tags are unmatched and should be removed. Also, the indentation looks weird on the webinterface (it suddenly changes from the nodes to the keys)

File: PQSMods/VertexColorMap.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+The `VertexColorMap { }` PQSMod is a mod that applies a color map over the terrain.
+
+**Example**
+PQS
+{
+	Mods
+	{
+		...
```
[the remark this answers is not in the document]
Comment (democat3457) [reply PQSMods/VertexColorMap.md]: What random indentation? The `...` indicates mods before and after VertexColorMap.

**Comment by StollD** [comment]
> What random indentation? The `...` indicates mods before and after
> VertexColorMap.

The mods node is indented differently than the Nodes inside of it. In the 
email it looks less weird than in the web interface, but I don't have 
the full file here.

Don't care too much about it. DeltaDizzy wanted me to review it so I 
just added everything I could think of. It just looked weird when 
viewing it with a browser, and the examples should be formatted with an 
unified style.

File: PQSMods/VertexColorMap.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (democat3457) [reply PQSMods/VertexColorMap.md]: > The mods node is indented differently than the Nodes inside of it. In the email it looks less weird than in the web interface, but I don't have the full file here.  
Isn't the mods node supposed to be indented differently than the nodes inside? The mods node is a wrapper for the nodes after all.

[the remark this answers is not in the document]
Comment (StollD) [reply PQSMods/VertexColorMap.md]: > Isn't the mods node supposed to be indented differently than the nodes
> inside? The mods node is a wrapper for the nodes after all.

Well, I am not talking about levels of indentation I am talking about how you get to those levels

When you look at it in the github web interface, it looks like this:
```
PQS
{
 Mods
 {
        VertexColorMap
        {
        }
 }
}
```

Which is most likely some weird tabs vs spaces conflict. Mods *should* be on its own indentation level, but it should use the same method of indentation that is used somewhere else (tabs, spaces, etc.). Maybe it is also just the web interface going crazy, it does look ok when you open the file as raw. Feel free to just ignore it.

About those `...`, I misunderstood them as code tags. You should probably add code tags then

File: main/ScaledVersion/OnDemand.md (hunk 1 above)
Comment (StollD) [reply main/ScaledVersion/OnDemand.md]: Ignore that, they are just dots. Hard to see on a smaller display.

File: PQSMods/VertexColorMap.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (democat3457) [reply PQSMods/VertexColorMap.md]: Yeah, pushing the code tags in my fork.

File: main/ScaledVersion/Coronas.md [hunk 3]
```diff
@@ -1,4 +0,0 @@
----
```
[the remark this answers is not in the document]
Comment (democat3457) [reply main/ScaledVersion/Coronas.md]: The page is/should be moved to Corona.md to be consistent with the wrapper nodes. (Biome.md)

[the remark this answers is not in the document]
Comment (StollD) [reply main/ScaledVersion/Coronas.md]: You still link to it on the main page:

`- [Coronas subnode]({{ "main/ScaledVersion/Coronas.html" | relative_url  }})`

[the remark this answers is not in the document]
Comment (democat3457) [reply main/ScaledVersion/Coronas.md]: huh. Well, it's fixed on my fork.

**Pull request merged**